src/dms-integration/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { MetadataProperty } from '../types';
+import { IDmsDynamicFormField } from './types';
+import { IBulkField } from '../features/bulk/bulk/store/model';
+
+function convertStringToKey(string: string): string {
+	return string.toLowerCase().replace(' ', '_');
+}
+
+export function convertDmsDynamicFormFieldsToMetadataProperty(fields: IDmsDynamicFormField[]): MetadataProperty[] {
```

Comment:
This is a tad ugly IMO. I think a much better approach would be to write this using function composition and avoid the if statements.

Either way like I stated in an earlier comment. This code really doesn't belong here.

Reply:
This code maybe doesn't belong here, but at the moment I do use it to mock a dms-integration in App.tsx.
I think it has value to have this as an example integration. Everything in this folder is only used to the App.tsx example integration.